File: app/src/main/java/com/callmangement/firebase/FirebaseUtils.java
package com.callmangement.firebase;


import com.google.firebase.messaging.FirebaseMessaging;

public class FirebaseUtils {
    public static void registerTopic(String topic){
        FirebaseMessaging.getInstance().subscribeToTopic(topic);
    }

    public static void unregisterTopic(String topic){
        FirebaseMessaging.getInstance().unsubscribeFromTopic(topic);
    }
}

File: app/src/main/java/com/callmangement/firebase/FirebaseUtils.kt
package com.callmangement.firebase

import com.google.firebase.messaging.FirebaseMessaging


object FirebaseUtils {
    fun registerTopic(topic: String) {
        FirebaseMessaging.getInstance().subscribeToTopic(topic)
    }

    fun unregisterTopic(topic: String) {
        FirebaseMessaging.getInstance().unsubscribeFromTopic(topic)
    }
}
